packages/input-iban/src/validators.js

```diff
@@ -153,3 +153,39 @@ export class IsCountryIBAN extends IsIBAN {
     return localize.msg('lion-validate+iban:error.IsCountryIBAN', data);
   }
 }
+
+export class IsNotCountryIBAN extends IsIBAN {
+  static get validatorName() {
+    return 'IsNotCountryIBAN';
+  }
+
+  /**
+   * @param {?} [value]
+   * @returns {Boolean}
+   */
+  execute(value) {
+    const notIBAN = super.execute(value);
+    if (value.slice(0, 2) === this.param) {
+      return true;
+    }
+    if (notIBAN) {
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * @param {object} [data]
+   * @param {*} [data.modelValue]
+   * @param {string} [data.fieldName]
+   * @param {*} [data.params]
+   * @param {string} [data.type]
+   * @param {Object.<string,?>} [data.config]
+   * @param {string} [data.name]
+   * @returns {Promise<string|Node>}
+   */
+  static async getMessage(data) {
+    await loadTranslations();
+    return localize.msg('lion-validate+iban:error.IsBlaklistedCountryIBAN', data);
```

Comment:
I would also change the name of the validation messages, makes it easier to find if it's the same name as the validator. 
```suggestion
    return localize.msg('lion-validate+iban:error.IsNotCountryIBAN', data);
```